internal/cmd/egctl/testdata/translate/out/default-resources.all.yaml

```diff
@@ -784,6 +784,14 @@ xds:
                     address: 1.1.1.1
                     portValue: 8000
               loadBalancingWeight: 1
+              metadata:
```

Comment:
shouldnt this be one level above ?
https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/endpoint/v3/endpoint_components.proto#envoy-v3-api-msg-config-endpoint-v3-localitylbendpoints

Reply:
Yes, it's more efficient and semantically correct to annotate the group of endpoints rather than each and every endpoint in that group.